locales/en/messages.json

```diff
@@ -5002,6 +5002,13 @@
     "osdDescUpDownReference": {
         "message": "OSD Symbol to show when pitch is approaching vertical (90 deg, U) and D for nose down (-90 deg, D)"
     },
+    "osdTextElementTxUplinkPower": {
+        "message": "Tx uplink power",
+        "description": "One of the elements of the OSD"
+    },
+    "osdDescTxUplinkPower": {
+        "message": "Shows a value of the Tx power (mw or W). Useful when <i>Dynamic Power</i> is enabled on the radio"
```

Comment:
Would suggest adding "for supporting radios" to avoid the expectation that this will work for all transmitter types.

Reply:
Done.